src/modules/CourseDetail/components/ReviewForm/index.tsx

```diff
@@ -60,10 +65,12 @@ export function ReviewForm() {
   /**
    * Use this function to cancel editing review
    */
-  function clearEditor() {
-    methods.setValue('content', INITIAL_CONTENT as TNode[])
-    setEditorValue(INITIAL_CONTENT)
-    methods.setValue('rating', 0)
+  function clearForm() {
+    methods.setValue('academicYear', defaultAcademicYear)
+    methods.setValue('semester', defaultSemester)
+    methods.setValue('rating', defaultRating)
+    methods.setValue('content', defaultContent)
```

Comment:
i think form can be reset using `reset(values)` from `useForm()`
[https://react-hook-form.com/api/useform/reset](https://react-hook-form.com/api/useform/reset)